xml/System.Net/FileWebRequest.xml

```diff
@@ -717,9 +717,29 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns a <see cref="T:System.IO.Stream" /> object for writing data to the file system resource as an asynchronous operation.</summary>
+        <returns>The Task object representing the asynchronous operation.</returns>
```

Comment:
"Task" -> `<see cref="...Task"/>`.

Reply:
@scalablecory @Jozkee there is a neat feature to help provide code suggestions that can be bulk merged:

```suggestion
        <returns>The <see cref="T:System.Threading.Tasks.Task" /> object representing the asynchronous operation.</returns>
```

Reply:
Note: The comment is already outdated so you may ignore it.